connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java

```diff
@@ -27,11 +27,13 @@
 import org.apache.kafka.connect.errors.NotFoundException;
 import org.apache.kafka.connect.runtime.AbstractHerder;
 import org.apache.kafka.connect.runtime.ConnectorConfig;
+import org.apache.kafka.connect.runtime.ConnectorStatus;
```

Comment:
Confronted with the choice of defining yet another private enum to represent state/action (START or STOP) and reusing some existing enum that is close to what I need, I chose for the moment the latter. But the former is pretty legit too. Let me know what you think.